compiler/modules/front.syntax/parseTypeDesc.bal

```diff
@@ -164,7 +164,8 @@ function parsePrimaryTypeDesc(Tokenizer tok) returns TypeDesc|err:Syntax {
         [IDENTIFIER, var identifier] => {
             Position pos = tok.currentStartPos();
             check tok.advance();
-            var [prefix, typeName] = check parseOptQualIdentifier(tok, identifier);
+            var [prefixNamePos, typeName] = check parseOptQualIdentifier(tok, identifier);
+            [string?, Position?] [prefix, varNamePos] = prefixNamePos == () ? [(), ()] : prefixNamePos;
```

Comment:
Make this more obvious.

varNamePos is not being used...

Reply:
fixed